devops/llvm-anybuild.yml

```diff
@@ -7,9 +7,26 @@ parameters:
   - name: BuildDebug
     type: boolean
     default: true
+  # extra command-line args to pass to AnyBuild
   - name: ExtraAnyBuildArgs
     type: string
-    default: ' '
+    default: '--Epoch 1'
+  # LLVM commit hash to build
+  - name: GOOD_HASH
+    type: string
+    default: '73811d3'
+  # args to pass to cmake when building
+  - name: CmakeBuildToolArgs
+    type: string
+    default: 'install'
```

Comment:
Having a default here means if some process overwrites this, the job won't install anymore unless the overwriting options have install in them. Maybe this could be empty and the usage below have a compulsory install in addition to any new options?

Reply:
My idea was exactly to be able to overwrite this when doing test runs, so that I can avoid doing the `install` step.

That's how it's already set up in the current pipeline; the difference is that in the current pipeline it's a variable that is still settable at queue time.

But I understand your concern and I can change it to make sure that `install` is performed.

Reply:
Ah, I see. No, if the idea is to not install, than this should be fine. I just wanted to make sure this was the semantics. Thanks!